src/views/TradingCompetition/index.tsx

```diff
@@ -58,11 +60,14 @@ const BottomBunnyWrapper = styled(Box)`
 `
 
 const TradingCompetition = () => {
+  // REINSTATE THIS CONST IN PROD
+  // const profileApiUrl = process.env.REACT_APP_API_PROFILE
+  const profileApiUrl = 'https://pancake-profile-api-git-develop-pancakeswap.vercel.app'
```

Comment:
You can just add `REACT_APP_API_PROFILE` to `.env.development.local`

Reply:
Ah much nicer cheers 👍